### PR TITLE
EPMRPP-90015 || Add possibility to configure autocomplete suggestions…

### PR DESCRIPTION
**Pull request by BlazarQSO**
… when inviting users

## PR Checklist

* [x] Have you verified that the PR is pointing to the correct target branch? (`develop` for features/bugfixes, other if mentioned in the task)
* [x] Have you verified that your branch is consistent with the target branch and has no conflicts? (if not, make a rebase under the target branch)
* [x] Have you checked that everything works within the branch according to the task description and tested it locally?
* [x] Have you run the linter (`npm run lint`) prior to submission? Enable the git hook on commit in your IDE to run it and format the code automatically.
* [x] Have you run the tests locally and added/updated them if needed?
* [x] Have you checked that app can be built (`npm run build`)?
* [x] Have you made sure that all the necessary pipelines has been successfully completed?
* [x] If the task requires translations to be updated, have you done this by running the `manage:translations` script?

### CODE REVIEW

**Review by Vadim73i — APPROVED** [review]

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx [hunk 1]
```diff
@@ -211,20 +221,37 @@ export class InviteUserModal extends Component {
...
-          id: 'externalUserInvitationModal',
-          data: { email: res.email, link: res.backLink },
-        });
+  inviteUserAndCloseModal = (closeModal) => async (data) => {
+    const { selectedProject, userSuggestions } = this.props;
+    const userData = {
+      ...data,
+    };
+
+    if (!(this.props.data.isProjectSelector || !userSuggestions)) {
```
Comment (AmsterGet) [inline app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx:230]: Optional. May be destructure the `isProjectSelector` from props?

File: app/src/controllers/appInfo/selectors.js [hunk 2]
```diff
@@ -65,6 +66,8 @@ export const instanceTypeSelector = (state) =>
   environmentSelector(state)[INSTANCE_TYPE] || NOT_PROVIDED;
 export const allowDeleteAccountSelector = (state) =>
   environmentSelector(state)[ALLOW_DELETE_ACCOUNT] === 'true';
+export const userSuggestionsSelector = (state) =>
```
Comment (AmsterGet) [inline app/src/controllers/appInfo/selectors.js:69]: Maybe rename it to match the boolean type?
F.e. `areUserSuggestionsAllowedSelector`.

File: app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx (hunk 1 above)
Comment (BlazarQSO) [reply app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx]: Done.

File: app/src/controllers/appInfo/selectors.js (hunk 2 above)
Comment (BlazarQSO) [reply app/src/controllers/appInfo/selectors.js]: Done.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**